lib/cdo/firehose.rb

```diff
@@ -57,6 +57,12 @@ class FirehoseClient
   # Ref: https://docs.aws.amazon.com/firehose/latest/dev/limits.html
   BYTES_PER_REQUEST = 1024 * 1024 * 4
 
+  # This limit is based on the empirical observation that no data_json column values larger than 65,526 bytes have
```

Comment:
We could note the exact length of the destination Redshift database columns where these 2 attributes get delivered.